pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -166,9 +166,14 @@ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBa
 			})
 		}
 		if len(addTagsInput.ResourceArns) > 0 && len(addTagsInput.Tags) > 0 {
-			_, err = c.elbv2.AddTags(addTagsInput)
-			if err != nil {
-				return nil, fmt.Errorf("Error adding tags after creating Load Balancer: %q", err)
+			// elbv2.AddTags doesn't allow to tag multiple resources at once
```

Comment:
I suggest replace the addTagsInput declaration with ResourceArns []*string, and []*elbv2.Tag{} directly, just to avoid the confusion to future ppl.

Reply:
Hi @M00nF1sh,

I'm not sure I understand what you mean.
Do you mean to use separate tags and arns variables prior to this code, and then build the `addTagsInput` only just before calling `AddTags` ?

Reply:
@M00nF1sh ,

I've pushed a new version that I think implements what you wanted.
Let me know if there's anything that needs to be changed :)

Thanks!

Reply:
Yes, that is exactly what i want 🤣 .

would you mind do a  slightly optimization to avoid memory realloc like below:
resourceArns := make([]*string, 0, len(mappings))
targetGroupTags := make([]*elbv2.Tag, 0, len(tags))

Reply:
My first implementation was like that :) then I thought it wasn't probably in a performance sensible code, so I wen't to the traditional way of declaring empty slices.
I'm going to push a revised version, thanks !

Reply:
I've pushed the modification. Thanks!